.github/workflows/phpunit-mysql.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['8.0', '8.1', '8.2']
-        server-versions: ['master']
+        server-versions: ['master', 'stable27']
```

Comment:
Why `stable27` this is only for the master branch, `stable27` should only be tested on its own branch.
(so backporting this to `stable27` instead)

Reply:
@susnux the original issue : https://github.com/nextcloud/logreader/issues/868 requested such?

Reply:
I think that is about adding 8.2 also to the `stable27` branch.

Reply:
yes, what susnux says